test/conformance/walk.go

```diff
@@ -209,11 +209,14 @@ func getConformanceDataFromStackTrace(fullstackstrace string) (*ConformanceData,
 		i += 2
 	}
 
-	// filenames have `/go/src/k8s.io` prefix which dont exist locally
-	for i, v := range frames {
-		frames[i].File = strings.Replace(v.File,
-			"/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes",
-			*k8sPath, 1)
+	// filenames are in one of two special GOPATHs depending on if they were
```

Comment:
even test binaries built with `rules_go` seem to have output here as if they've been `strip`ed, while the make build does not (even though we pass -ldflags etc.). 

I have some follow-up about how we handle that, but:
- I'm not sure we want to be always stripping e2e.test
- Even if we do, we need to discuss doing this in the modern go way with `-trimpath` toplevel flag instead of gc/ldflags, which do *not* have the same output or behavior (modern `-trimpath` replaces filepaths with import paths which would be a breaking change here and probably elsewhere, it's more portable and actually supported by go for reproducible builds, but ... that should be another PR/discussion)

Reply:
+1 to using standard stuff like trimpath!

Reply:
as a follow up.